src/material/menu/menu-trigger.ts

```diff
@@ -282,7 +282,7 @@ export class MatMenuTrigger implements AfterContentInit, OnDestroy {
   }
 
   /** Opens the menu. */
-  openMenu(): void {
+  openMenu(disableAutoFocus?: boolean): void {
```

Comment:
do we actually want to add it to the public API? why don't we just make a private `_openMenu(autofocus)` and then define `openMenu` as `_openMenu(true)`

Reply:
I thought it could be useful if opening programmatically as well. It seems like there might be other times where one wants to just open the menu but not force the focus to be shifted within the open menu.

But we could hide it to minimize the api footprint if it is hasn't come up.

Reply:
Tweaked. I flipped the boolean too since now it can just be explicitly passed in only where needed.